packages/rizin/build.sh

```diff
@@ -0,0 +1,9 @@
+# Thank you @TomJo2000
+TERMUX_PKG_HOMEPAGE=https://rizin.re/
+TERMUX_PKG_DESCRIPTION="UNIX-like reverse engineering framework and command-line toolset. (fork of radare2)"
```

Comment:
I think we can remove the `(fork of radare2)` comment, the project should hopefully be recognizable on its own